src/clojure_turtle/core.cljc

```diff
@@ -231,28 +230,39 @@
 (defn draw
   "The function passed to Quil for doing rendering."
   []
-  (q/with-translation [(/ (q/width) 2) (/ (q/height) 2)]
-    (reset-rendering)
-    
-    (q/push-matrix)
-    (q/apply-matrix 1  0 0
-                    0 -1 0)
-    (doseq [l @lines]
-      (let [[[x1 y1] [x2 y2]] l]
-        (q/line x1 y1 x2 y2))) 
-    (draw-turtle) 
-    (q/pop-matrix)))
+  (q/push-matrix)
+  (q/translate (/ (q/width) 2) (/ (q/height) 2))
+  (reset-rendering)
+  (q/push-matrix)
+  (q/scale 1.0 -1.0)
```

Comment:
The **apply-matrix** and **with-translation** function/macro both aren't available for clojurescript. So I replaced them with the equivalent(?) q/translate and q/scale.

The result should be identical, so this diff is for cljs compatibility only.